<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsComponent</class>
 <widget class="QTabWidget" name="CSettingsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_SettingsTrafficNetworkServers">
   <attribute name="title">
    <string>Servers</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsTrafficNetworkServers">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsNetworkServersComponent" name="comp_SettingsServersComponent">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vs_SettingsTrafficNetworkServers">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsTrafficNetwork">
   <attribute name="title">
    <string>Network</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsTrafficNetwork">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsNetworkComponent" name="comp_SettingsNetwork">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsAudio">
   <attribute name="title">
    <string>Audio</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsAudio">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CAudioSetupComponent" name="comp_AudioSetup">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsSimulator">
   <attribute name="title">
    <string>Simulator</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsSimulator">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsSimulatorComponent" name="comp_SettingsSimulator">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsSimulatorBasics">
   <attribute name="title">
    <string>Simulator (Basics)</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsSimulatorBascis">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsSimulatorBasicsComponent" name="comp_SettingsSimulatorBasics">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vs_SettingsSimulatorBasics">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsSimulatorMessages">
   <attribute name="title">
    <string>Simulator (messages)</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsSimulatorMessages">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsSimulatorMessagesComponent" name="comp_SettingsSimulatorMessages">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vs_SettingsSimulatorMessages">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsGui">
   <attribute name="title">
    <string>GUI</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsGui">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QFormLayout" name="fl_SettingsGui">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="gb_GuiGeneral">
        <property name="title">
         <string>General</string>
        </property>
        <layout class="QFormLayout" name="fl_GuiGeneral">
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiOpacity">
           <property name="text">
            <string>Opacity (0-100%)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="hs_SettingsGuiOpacity">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cb_SettingsGuiWidgetStyle"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiWidgetStyle">
           <property name="text">
            <string>Widget Style</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="gb_Font">
        <property name="title">
         <string>Font</string>
        </property>
        <layout class="QFormLayout" name="fl_GuiFont">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiFont">
           <property name="text">
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFontComboBox" name="cb_SettingsGuiFont">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiFontSize">
           <property name="text">
            <string>Font size (pt)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_SettingsGuiFontSize">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiFontStyle">
           <property name="text">
            <string>Font style</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cb_SettingsGuiFontStyle">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bold</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>italic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bold italic</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiFontColor">
           <property name="text">
            <string>Font color</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QWidget" name="wi_FontColor" native="true">
           <layout class="QHBoxLayout" name="hl_FontColor">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="le_SettingsGuiFontColor">
              <property name="maxLength">
               <number>200</number>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_SettingsGuiFontColor">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="gb_ViewRefreshTimes">
        <property name="title">
         <string>View refresh times</string>
        </property>
        <layout class="QFormLayout" name="fl_ViewRefreshTimes">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiAircraftRefreshTime">
           <property name="toolTip">
            <string>Aircraft refresh time (5-30s)</string>
           </property>
           <property name="text">
            <string>Aircraft rt (5-30s)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="hs_SettingsGuiAircraftRefreshTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiAtcRefreshTime">
           <property name="toolTip">
            <string>ATC refresh time (5-30s)</string>
           </property>
           <property name="text">
            <string>ATC rt (5-30s)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="hs_SettingsGuiAtcRefreshTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_SettingsGuiUserRefreshTime">
           <property name="toolTip">
            <string>User refresh time (5-30s)</string>
           </property>
           <property name="text">
            <string>User rt (5-30s)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="hs_SettingsGuiUserRefreshTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>lbl_SettingsGuiUserRefreshTime</zorder>
        <zorder>lbl_SettingsGuiAtcRefreshTime</zorder>
        <zorder>lbl_SettingsGuiAircraftRefreshTime</zorder>
        <zorder>hs_SettingsGuiUserRefreshTime</zorder>
        <zorder>hs_SettingsGuiAtcRefreshTime</zorder>
        <zorder>hs_SettingsGuiAircraftRefreshTime</zorder>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QWidget" name="qw_SettingsGuiFont" native="true">
        <layout class="QHBoxLayout" name="hl_SettingsGuiFontColor">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsHotkeys">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="title">
    <string>Hotkeys</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsMisc">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsHotkeyComponent" name="comp_SettingsHotkeysComponent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsAdvanced">
   <attribute name="title">
    <string>Advanced</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="BlackGui::Components::CSettingsAdvancedComponent" name="comp_SettingsAdvancedComponent">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CAudioSetupComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/audiosetupcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsAdvancedComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsadvancedcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsSimulatorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingssimulatorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsNetworkServersComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsnetworkserverscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsNetworkComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsnetworkcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsHotkeyComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingshotkeycomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsSimulatorBasicsComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingssimulatorbasicscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsSimulatorMessagesComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingssimulatormessagescomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
